ModulBot/Commands/TuneAdmin.cs

```diff
@@ -0,0 +1,117 @@
+﻿using ModulBot.Database.PostgresRepositories;
+using ModulBot.Interfaces;
+using ModulBot.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using ModulBot.Attributes;
+
+namespace ModulBot.Commands
+{
+    [NotShowInHelp]
+    [Serializable]
+    public class TuneAdmin : ITool
+    {
+        public string Description { get; set; }
+        public List<string> CommandsName { get; set; }
+        public bool IsAdmin { get; set; }
+        private static int step = 0;
+        private static int slctUserId = 0;
+
+        public async void Run(Message message)
+        {
+            var db = new PostgresUserRepository();
+            StringBuilder allUsers = new StringBuilder();
+            foreach (var user in db.GetAll().OrderBy(x => x.Id))
+            {
+                if (user.ChatId > 0) // не берем групповые чаты
+                {
+                    string _isAdmin = user.IsAdmin ? "admin" : "user";
+                    allUsers.Append($"{user.Id}. {user.FirstName} - ({_isAdmin})\n");
+                }
+            }
+            if (message.Text[0] == '/')
+            {
+                message.Text = message.Text.Substring(1);
+            }
+
+            switch (step)
+            {
+                case 0:
+                    if (message.Chat.Id < 0) //в групповых чатах бот не видит обычных сообщений
+                    {
+                        await Bot.BotClient.SendTextMessageAsync(message.Chat.Id, "*Внимание!* В групповых" +
+                            " чатах перед ответом ставьте слэш \"*/*\", " +
+                            "т.к. бот не видит обычных сообщений.\nНапример: */да*, */1*, и т.д.",
+                            Telegram.Bot.Types.Enums.ParseMode.Markdown);
+                    }
+
+                    Bot.IsSetAdminStart = true;
+
+                    await Bot.BotClient.SendTextMessageAsync(message.Chat.Id, "Вот список всех" +
+                        " пользователей. Отправьте номер пользователя, права которого" +
+                        " вы хотите настроить\n" + allUsers);
+                    step++;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this set, which updates a 'static' field from an instance method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2696)